cherrypy/test/test_static.py

```diff
@@ -97,6 +97,22 @@ def bytesio(self):
                 f = io.BytesIO(b'Fee\nfie\nfo\nfum')
                 return static.serve_fileobj(f, content_type='text/plain')
 
+            @cherrypy.expose
+            def serve_file_utf8_filename(self):
+                file_path = os.path.join(curdir, 'style.css')
+                return static.serve_file(
+                    file_path,
+                    disposition='attachment',
+                    name='has_utf-8_character_☃.html')
+
+            @cherrypy.expose
+            def serve_fileobj_utf8_filename(self):
+                f = open(os.path.join(curdir, 'style.css'), 'rb')
+                return static.serve_fileobj(
+                    f,
```

Comment:
Let's not hit the disk here:
```suggestion
                return static.serve_fileobj(
                    io.BytesIO(b'☃\nfie\nfo\nfum'),
```